app/jobs/notify_web_hook_job.rb

```diff
@@ -3,8 +3,11 @@ class NotifyWebHookJob < ApplicationJob
   include TraceTagger
 
   queue_as :default
+  queue_with_priority PRIORITIES.fetch(:web_hook)
```

Comment:
:thinking: Seems it makes sense to move `PRIORITIES` from `delayed_job.rb` initializer into application.rb or something generic. I can do that in followup PR.